tools/notifyServicelogEvent.c

```diff
@@ -177,18 +169,21 @@ main(int argc, char *argv[])
     }
 
   done:
+    g_strfreev(processed_args);
+    pcmk__free_arg_context(context);
+
     if (event != NULL) {
         servicelog_event_free(event);
     }
 
     if (slog != NULL) {
         servicelog_close(slog);
     }
-
     closelog();
 
     if (exit_code == CRM_EX_OK) {
         exit_code = pcmk_rc2exitc(rc);
     }
+    pcmk__output_and_clear_error(error, NULL);
     crm_exit(exit_code);
 }
```

Comment:
Does notifyServicelog need a man page?

Reply:
Maybe? What do I need to add it to to make that happen? I don't see the other tools listed explicitly anywhere so I figured it would get picked up as part of `sbin_PROGRAMS`.